pkg/apis/mongodb/v1/mongodb_types.go

```diff
@@ -298,6 +298,30 @@ func (m MongoDB) GetFCV() string {
 	return strings.Join(parts[:minorIndex+1], ".")
 }
 
+func (m MongoDB) DesiredReplicas() int {
+	return m.Spec.Members
+}
+
+func (m MongoDB) CurrentReplicas() int {
+	return m.Status.CurrentStatefulSetReplicas
+}
+
+func (m *MongoDB) StatefulSetReplicasThisReconciliation() int {
+	return scale.ReplicasThisReconciliation(m)
+}
+
+type automationConfigReplicasScaler struct {
+	current, desired int
+}
+
+func (a automationConfigReplicasScaler) DesiredReplicas() int {
+	return a.desired
+}
+
+func (a automationConfigReplicasScaler) CurrentReplicas() int {
+	return a.current
+}
```

Comment:
re-use existing interface to allow us to get the correct number of automation config members each reconciliation (we will get all of them on first creation, otherwise we scale up/down one at a time)